space_age/.byebug_history

```diff
@@ -1,4 +1,66 @@
 exit
+Destination.find_by(planet: "neptune")
+params[:planet]
+params
+exit
```

Comment:
I think this might just come with using the byebug gem but I don't think it's a big deal